.github/workflows/release.yml

```diff
@@ -40,7 +40,8 @@ jobs:
 
       - name: Build Project
         env:
-          REPR_GUEST_BUILD: 1
+          # TODO: make this dynamic somehow
```

Comment:
this does not matter for reasons explained on huddle, remove comment and SKIP_GUEST_BUILD in this build